tests/unit/test_cli.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+import tempfile
+
+import unittest
+from unittest import mock
+
+import hotsos.cli
+
+
+class TestCLI(unittest.TestCase):
+
+    def test_get_hotsos_root(self) -> None:
```

Comment:
does the method annotations have a purpose here?

Reply:
I mean it's just a habit of mine. We could eventually start checking using [mypy](http://mypy-lang.org/)

Reply:
If we merge canonical/hotsos#494 we can start statically type checking the code.